app/views/posts/index.html.erb

```diff
@@ -3,24 +3,27 @@
       <div class="user_details">
         <h3><%= @user.name %></h3>
         <p>Number of posts: <%=@user.post_counter%>
+        <%= Link_to "Add a post", new_user_post_path(@user) %>
```

Comment:
- [ ] I am getting error when clicking on any posts. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/179086631-67fd51e2-8233-4b1d-b72a-66f68a6edf48.png)

- [ ] Also see all post button giving error too after clicking on it.

![image](https://user-images.githubusercontent.com/38635158/179086755-8bd83ec8-eb40-4303-9ccb-d1295e75ccc3.png)